spec/requests/v6/assessments_controller/simple_spec.rb

```diff
@@ -513,64 +520,145 @@ module V6
       end
 
       context "with dependants" do
-        let(:params) { { dependants: dependant_params } }
+        context "with no income for dependants" do
+          let(:params) { { dependants: dependant_params.map { _1.except(:income) } } }
 
-        it "returns a version of 6" do
-          expect(parsed_response.fetch(:version)).to eq("6")
+          it "creates a log record" do
```

Comment:
This works, but feels a bit strange - we don't want to check the log record (note the redactions in the DOBs, which are not obvious) - would be better (and more idiomatic) to check the response rather than the log of the response